src/main/docker/gitlab/gitlab-config.rb

```diff
@@ -0,0 +1,75 @@
+## GitLab configuration settings
```

Comment:
Instead of providing the whole file you may want to add something like that to the docker-compose environment:

```
 environment:
      - GITLAB_OMNIBUS_CONFIG=prometheus_monitoring['enable'] = false; nginx['listen_port'] = 80; gitlab_rails['monitoring_whitelist'] = ['0.0.0.0/0']

```

See also https://github.com/kit-sdq/Artemis-SAML2-Test-Docker/blob/main/docker-compose.yml#L29

Reply:
Ah that's interesting. I think the reason why I opted for a file, is so that the user can easily extend it in case he/she wants to add more options. Do you think the file is a bit of an overkill (I like your suggestion since we avoid creating an extra file. ?

Reply:
In addition, you won't get problems with newly added configuration values. If you provide a file, you have to make sure that this file is up to date according to the latest version of gitlab .. forever :) . If you do not provide a file. You have no need to update it again and again.

Reply:
That's a valid point. I will definitely change that!